src/flicking.js

```diff
@@ -282,24 +282,27 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		_setPadding: function(padding, build) {
 			var horizontal = this.options.horizontal;
 			var panel = this._conf.panel;
+			var paddingSum = padding[0] + padding[1];
+			var cssValue = {};
 
 			panel.size = this.$wrapper[
-				horizontal ? "outerWidth" : "height"
-			]() - (padding[0] + padding[1]);
+					horizontal ? "innerWidth" : "innerHeight"
+				]() - paddingSum;
 
-			var cssValue = {
-				padding: (horizontal ?
-				"0 " + padding.reverse().join("px 0 ") :
-				padding.join("px 0 ")) + "px"
-			};
+			if (paddingSum || !build) {
+				cssValue.padding = (horizontal ?
+						"0 " + padding.reverse().join("px 0 ") :
+						padding.join("px 0 ")) + "px";
+
+				cssValue[ horizontal ? "width" : "height" ] = panel.size;

```

Comment:
This line solves problem.
But how about apply
```
box-sizing: border-box; 
```
style rule to the wrapper. This also solves problem.
And it doesn't set pixel based size of the element. so it will works better on responsive pages.